### ISSUE TITLE
release packetdrill

### ISSUE THREAD

**Issue by ligurio** [nplab/packetdrill#147]
Right now repo contains only source in git and it is not clear how to get a stable version.
Please consider publishing packetdrill archives as github releases.
See https://help.github.com/en/github/administering-a-repository/creating-releases

**Comment by tuexen** [comment]
Making releases would add confusion between the official packetdrill repo / releases and what is done in this fork (adding support for FreeBSD, Mac OS X, Solaris, UDP Lite, SCTP).

**Comment by ligurio** [comment]
Yeah, I know. But you are already forked it and it's anyway confuse users. With releases it would be easy to rely on tool.
By the way do you plan to merge your changes to upstream?

// CC: @nealcardwell

**Comment by tuexen** [comment]
Before forking, I tried to contribute for over a year or so. Since that did not work out and I needed an extended version, I forked. This forking happened before Googles packetdrill sources moved to GitHub.

I would be happy to give up the fork, but adding

- FreeBSD support
- MacOS support
- Solaris support
- UDPLite support
- SCTP support

might not of interest to Google and would it not be easy for me to dissect the changes to provide simple patches.

**Comment by nealcardwell** [comment]
Yes, I'm afraid I was a bottleneck, due to me not having enough cycles for
reviewing and integrating into Google's packetdrill sources. (Packetdrill
is not really my "day job', BBR and TCP are...).

neal

**Comment by ligurio** [comment]
@nealcardwell what about merging these features in the near future? I can help with testing patches.
